### PR TITLE
Update: Survey and Observation Vocabularies

### PR DESCRIPTION
**Pull request by nicholascar**
Survey Method & Type just renamed to Observation Method & Type for a starting point.

Survey Detail Type, seemingly a test vocab, removed.

Some small SHACL fixes too.

### CODE REVIEW

**Comment by KellyVance** [comment]
Survey_detail_type is a placeholder rather than a test (provides a reference point for Devs to point to). It will be populated with proper content in the future.

I'm uncomfortable approving this just yet. I think in the context of the discussions that were had we need to holistically look at survey types, survey methods, observation types, observation methods, and observation instruments/apparatus before we commit to changes that may have unintended consequences

**Comment by nicholascar** [comment]
> survey_detail_type is a placeholder...

Sure, but it shouldn't be in `master` branch or else it will be picked up by the VocPrez loader script. So it should just live in a branch as a placeholder until it's ready for production.

Gotta keep that `master` clean.

> I'm uncomfortable approving this just yet.

Sure, just have the PR here and we can edit it (anyone can edit the branch) until it's ready

**Comment by KellyVance** [comment]
closes #206 
closes #188

**Comment by KellyVance** [comment]
@nicholascar 
Merged #219 into this PR. 
Addresses Survey Type and Method holistically with Observation Type and Method. Can you please comment on changes as needed?

@Greenwoodmatthew @CantRoger these are the changes to survey and obs as discussed.

As always these are just an updated edition to be augmented and added to as needs arise.

File: vocabularies/survey-type.ttl [hunk 1]
```diff
@@ -172,128 +83,23 @@ svytp:physical-geology a skos:Concept ;
...
-    skos:notation "gv" ;
-    skos:prefLabel "Gravity"@en ;
-    skos:topConceptOf <http://linked.data.gov.au/def/survey-type> .
-
-svytp:magnetic a skos:Concept ;
-    skos:altLabel "aeromagnetics"@en ;
-    skos:definition "A survey where the variations of the Earth's magnetic field is measured to determine the distribution of magnetic minerals (predominately magnetite)"@en ;
+    
+svytp:spectral a skos:Concept ;
+    skos:altLabel "hyperspectral"@en ;
```
Comment (Greenwoodmatthew) [inline vocabularies/survey-type.ttl:101]: add new altlabel: multispectral

Comment (geoderekh) [reply vocabularies/survey-type.ttl]: Spectral needs a notation. 
Also, Idk about the logic behind using altlabels as opposed to narrower terms for multispectral (aster) and hyperspectral (hymap).  Without these as narrower terms. how can I facet my search results to weed out the things I don't want? I see that hyper and multi are here as observation_types. Do we have a ckan profile for observation, or is the obs_type somewhere on the survey profile?

Comment (KellyVance) [reply vocabularies/survey-type.ttl]: I'll add skos:notation "sp" ;
I have the altLabels there. But I'm not super comfortable having them as double-ups with the observation types. Really it should just be Survey = spectral, observation = hyperspectral
where you'd facet your search at the observation level. same as how you'd distinguish between a mag and a grav survey that are both just geophysical at the survey level.

https://github.com/geological-survey-of-queensland/gsq-observation-profile/tree/review_observation_profile

Comment (Greenwoodmatthew) [reply vocabularies/survey-type.ttl]: my 2 cents on this is that like geophysics we don't care at the survey level how many bands they are observing.   Thats all obs type/method stuff.  The alt label was added as people refer to all spectral surveys often as hyperspectral regardless of the type of survey

Comment (CantRoger) [reply vocabularies/survey-type.ttl]: @geoderekh Yep I've already let David know we'll need a 'Spectral' CKAN profile based off the vanilla 'grav' survey profile. Observation Method/Instrument fields should enable people to refine searches for Multi/Hyper data depending on where they've been included in the Vocabs.

**Comment by KellyVance** [comment]
@nicholascar can you please run SHACL validation over the new file additions and generally pass your eye over the changes?

**Comment by KellyVance** [comment]
Pull request 25 days old without approval, resulting in parallel updates and conflicts with master branch. relevant files to be updated in new pull request.

**Pull request closed without merging**